src/Germany/Mews.Fiscalizations.Germany/V2/Model/ClientState.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Mews.Fiscalizations.Germany.V2.Model
+{
+    public enum ClientState
+    {
+        Enabled,
+        Disabled
```

Comment:
Is there a reason to use different naming than the API? Considering the users of the lib will need to read documentation to understand the workflow, wouldn't it be easier to have the naming consistent with the API and official docs?

Reply:
This model is what will be used on the client side (Mews in our case), In the DTO, they are called just like in Fiskaly documentation.

Reply:
I guess this is a general point on how we want to work with naming in the libraries. But as long as the original names are reasonable, I'd keep using those. We know that different naming in the lib vs the service causes confusion in some cases.